### DISCUSSION TITLE
[Feature request] ability to expose html report via network

### DISCUSSION THREAD

**Discussion by m3dbedb** [DannyDainton/newman-reporter-htmlextra#417] [Ideas]
The project allows to generate test report, but how do I expose it over network (e.g. 80-th port)

**Comment by w4dd325** [comment]
Where and how you run your test/report won't be to do with the code, it
will be to do with the pipeline you configure.

Personally mine is configured in github workflow. The test runs on a github
runner (configured using yml files) and stores the report as an artifact on
completion.
